From 475d3e782f84beec9ce7b4557f2444df0e2f914f Mon Sep 17 00:00:00 2001
From: Hannes Reinecke <hare@suse.de>
Date: Wed, 16 Apr 2014 13:30:47 +0200
Subject: fcoe.service: Add '--foreground' to prevent fcoemon to be killed

fcoemon is running as a daemon per default, so when using
Type=simple in the service file systemd will kill the fcoemon
daemon immediately as it just sees the return code from the
first fork() call.

References: bnc#873269

Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 etc/systemd/fcoe.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/systemd/fcoe.service b/etc/systemd/fcoe.service
index 2e10bcd..b1d9567 100644
--- a/etc/systemd/fcoe.service
+++ b/etc/systemd/fcoe.service
@@ -6,7 +6,7 @@ After=syslog.target network.target
 Type=simple
 EnvironmentFile=/etc/fcoe/config
 ExecStartPre=/sbin/modprobe -qa $SUPPORTED_DRIVERS
-ExecStart=/usr/sbin/fcoemon --debug=$DEBUG --syslog=$SYSLOG
+ExecStart=/usr/sbin/fcoemon --foreground --debug=$DEBUG --syslog=$SYSLOG
 
 [Install]
 WantedBy=multi-user.target
-- 
1.8.4.5

